src/tmx/Asn_J2735/src/r63/TestMessage00.c

```diff
@@ -40,7 +39,7 @@ asn_SEQUENCE_specifics_t asn_SPC_TestMessage00_specs_1 = {
 	offsetof(struct TestMessage00, _asn_ctx),
 	asn_MAP_TestMessage00_tag2el_1,
 	2,	/* Count of tags in the map */
-	asn_MAP_TestMessage00_oms_1,	/* Optional members */
+	0, 0, 0,	/* Optional elements (not needed) */
```

Comment:
@dan-du-car  Can you explain why these were included in the first place and why they are no longer necessary?